mappings/net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket.mapping

```diff
@@ -1,10 +1,9 @@
 CLASS net/minecraft/class_2797 net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket
 	COMMENT A packet used to send a chat message to the server.
 	COMMENT
-	COMMENT <p>This truncates the message to at most {@value #MAX_LENGTH} characters
-	COMMENT before sending to the server on the client. If the server receives the
-	COMMENT message longer than {@value #MAX_LENGTH} characters, it will reject
-	COMMENT the message and disconnect the client.
+	COMMENT <p>This truncates the message to at most 256 characters before sending to
```

Comment:
Is the constant moved to packetbytebuf? curious.

Reply:
Did not check, I believe CommandExecutionC2S had one. Unsure if mojang reuses that or if there are consts in other, more reasonable places.

Reply:
Checked constants file, the only `256` in networking-related stuff is in CEC2S. No new const in PacketByteBuf.